tests/utils/test_helpers.py

```diff
@@ -254,6 +255,40 @@ def test_validate_tag_name(tag_name, expected):
         )
 
 
+@pytest.mark.parametrize(
+    "artifact_name,expected",
+    [
```

Comment:
put each valid / invalid under blocks of "valid names" / "invalid names", could be easier when reading
and forcing developers to add cases to correct place, not making them mixed